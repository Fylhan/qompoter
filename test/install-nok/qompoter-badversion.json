{
	"name": "test/install-nok-badversion",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"cp/fylhan": "v0.1",
		"git/acme": "dev-master"
	}
}
