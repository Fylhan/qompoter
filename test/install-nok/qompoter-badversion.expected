Qompoter
======== install

* cp/fylhan v0.1
  Downloading sources...
  Error: no package found 'qompoter-repo/cp/fylhan/v0.1'
  FAILLURE

FAILLURE
